pages/common/kill.md

```diff
@@ -23,6 +23,14 @@
 
 `kill -{{9|KILL}} {{process_id}}`
 
-- Signal the operating system to pause a program, it until a SIGCONT ("continue") signal is received:
+- Signal the operating system to pause a program until a SIGCONT ("continue") signal is received:
 
 `kill -{{17|STOP}} {{process_id}}`
+
+- Send a signal to all processes with the given GID (group id):
+
+`kill -{{signal_name}} -{{group_id}}`
+
+- Kill every process that can be killed by the current user (except init and kill itself):
```

Comment:
IMO, I don't think this example is necessary in a tldr page. Let us remove this.